packages/app/lib/src/features/home/data/post_repository.dart

```diff
@@ -24,6 +24,13 @@ Have flutter access user location
 
 abstract interface class PostRepository {
   //Have funcs in heres
+
+  // TODOadd in get funcs
+
+  final List<Document> _posts = [];
+
+  //get all todos func for other classes
+  List<Document> get allPosts => _posts;
```

Comment:
You really don't need this. Repos have funcs. Services have state like this. We can talk about it on Thurs.